src/libraries/Common/tests/System/Net/EnterpriseTests/setup/linuxclient/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+cp /SHARED/linuxclient.keytab /etc/krb5.keytab
+
+# Keep the container running
+tail -f /dev/null
```

Comment:
this looks strange. often you can simply run bash so when executed interactively you get shell.

Reply:
Yes, it is strange. :) But I got the tip from looking at different containers everywhere.

I use it because 'docker-compose up -d' keeps the cluster of machines running in the background. And specifying only 'bash' as the EXEC/CMD of the container doesn't work for detached containers. It would cause the container to exit immediately.

The workflow is that once the containers are up and running in a detached mode, then one can access the containers by doing "docker exec -it linuxclient bash" (for example). That gives you a bash session where you are equivalently "logged on" to the container much like SSH. And then doing 'exit' from that drops you back to your host machine, but keeps the container running (as designed).

When you are done with all the containers, doing a 'docker-compose down' shuts them all down and removes the private network.

I documented this workflow in the README.md.